test/users.test.js

```diff
@@ -0,0 +1,156 @@
+              expect(res.body).to.haveOwnProperty('errors')
+                .to.haveOwnProperty('message')
+                .to.equal('Your account has already been verified');
+              done();
+            });
+        });
+    });
+    it('Should verify a user\'s account if the token is valid and the account hasn\'t been verified', (done) => {
+      User.findOne({ where: { email: mockData.user1.email } })
+        .then((user) => {
+          const token = tokenService.generateToken({ id: user.id }, 60 * 20);
+          return token;
+        })
+        .then((token) => {
+          chai.request(server)
+            .get(`${baseUrl}verify/${token}`)
+            .end((err, res) => {
+              expect(res.status).to.equal(200);
+              expect(res.body).to.haveOwnProperty('message')
+                .to.equal('Your account has been verified');
+              done();
+            });
+        });
+    });
+  }); describe('Requesting for verification mail resend on route(/api/user/verify/resend-email)', () => {
+    it('Should not resend email if email was not provided', (done) => {
+      chai.request(server)
+        .post(`${baseUrl}verify/resend-email`)
+        .end((err, res) => {
+          expect(res.status).to.equal(400);
+          expect(res.body).to.haveOwnProperty('errors')
+            .to.haveOwnProperty('message')
+            .to.equal('Email not provided');
+          done();
+        });
+    });
+    it('Should not resend email if provided email is not a valid email', (done) => {
+      chai.request(server)
+        .post(`${baseUrl}verify/resend-email`)
+        .send({
+          email: 'not-a-valid-email'
+        })
+        .end((err, res) => {
+          expect(res.status).to.equal(400);
+          expect(res.body).to.haveOwnProperty('errors')
+            .to.haveOwnProperty('message')
+            .to.equal('Please provide a valid email');
+          done();
+        });
+    });
+    it('Should not resend email if provided email dose not exist in the database', (done) => {
+      chai.request(server)
+        .post(`${baseUrl}verify/resend-email`)
+        .send({
+          email: faker.internet.email()
+        })
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          expect(res.body).to.haveOwnProperty('errors')
+            .to.haveOwnProperty('message')
+            .to.equal('User not found in the database');
+          done();
+        });
+    });
+    it('Should resend an email if the user with the email exists and the account hasn\'t been verified', (done) => {
+      chai.request(server)
+        .post(`${baseUrl}verify/resend-email`)
+        .send({
+          email: mockData.user3.email
+        })
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body).to.haveOwnProperty('message')
+            .to.equal('Your verification link has been resent');
+          done();
+        });
+    });
+  });
+});
```

Comment:
Everything seems fine. I will only add that you properly describe how your task can be manually tested in your PR description

Reply:
Done